pyigm/metallicity/mcmc.py

```diff
@@ -61,7 +61,7 @@
 import matplotlib as mpl
 
 #The next line works on queue systems for DISPLAY issues
-#mpl.use('Agg')   #  Commented out by X to avoid test failures
+mpl.use('Agg')
```

Comment:
This may cause test failures..
If it is required then we might need
to change the tests..